mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/constants/TelemetryConstants.java

```diff
@@ -10,7 +10,8 @@
 
 public class TelemetryConstants {
 
-  public static final String OPERATING_SYSTEM = "Android - " + Build.VERSION.RELEASE;
+  public static final String PLATFORM = "Android";
+  public static final String OPERATING_SYSTEM = PLATFORM + " - " + Build.VERSION.RELEASE;
```

Comment:
FWIW, @mick also specified the platforms as lowercased without spaces, e.g. (`android-x.x.x` and `ios-x.x.x`). We should circle back on this with @mick for consistency.